File: src/CopilotDiscordBot/Models/SessionRecord.cs
namespace CopilotDiscordBot.Models;

public sealed record SessionRecord(
    ulong ThreadId,
    ulong GuildId,
    ulong ParentChannelId,
    string CopilotSessionId,
    string? RepoPath,
    string ConfigJson,
    DateTimeOffset CreatedUtc,
    DateTimeOffset UpdatedUtc
);

File: src/models/SessionRecord.ts
export interface SessionRecord {
  threadId: bigint;
  guildId: bigint;
  parentChannelId: bigint;
  copilotSessionId: string;
  repoPath: string | null;
  configJson: string;
  createdUtc: Date;
  updatedUtc: Date;
}
